config/routes.rb

```diff
@@ -8,7 +8,7 @@
   get "/techhire-locations", to: "static_pages#techhire_locations", as: "techhire_locations"
   get "/download", to: "static_pages#download", as: "download"
   get "/companies/welcome", to: "companies#welcome"
-  get 'signup', to: "static_pages#signup", as: :signup
+  get 'signup', to: "companies#new", as: :signup
```

Comment:
serving `/signup` with `Companies#new` will allow us to consolidate those 2 actions and have improved rendering in the case when there are errors with a signup submission and we need to `render :new` so the user can correct them